src/solc_0.8/defi/rewardCalculation/TwoPeriodsRewardCalculator.sol

```diff
@@ -39,6 +40,8 @@ contract TwoPeriodsRewardCalculator is IRewardCalculator, AccessControl {
 
     event SavedRewardsSet(uint256 indexed reward);
 
+    IERC20RewardPool public rewardPoolInterface;
```

Comment:
Where is `rewardPoolInterface` initialize ?

Reply:
here -> https://github.com/thesandboxgame/sandbox-smart-contracts/blob/audit/staking-M-02/src/solc_0.8/defi/rewardCalculation/TwoPeriodsRewardCalculator.sol#L75